config/initializers/delayed_job_config.rb

```diff
@@ -10,6 +10,17 @@
 
 Delayed::Worker.delay_jobs          = !Rails.env.test?
 
+if ENV['HEROKU'].present?
+  Delayed::Worker.logger = Logger.new(STDOUT)
+else
+  Delayed::Worker.logger = Logger.new(File.join(Rails.root, 'log', 'delayed_job.log'))
+end
+
+if Delayed::Worker.delay_jobs && ($PROGRAM_NAME =~ /delayed_job/)
+  ActiveRecord::Base.logger = Delayed::Worker.logger
+  Rails.logger = Delayed::Worker.logger
```

Comment:
Right now `Rails.logger.info` message is not appearing in any BB application which is hosted anywhere if that code is executed by DJ. 

This is the only way I could make it work after two days trying various options and understanding how monit starts our DJ. 

If anyone can make it work in any other way then let's discuss.